joomla/joomla-cms
# [4] Remove unused com_media field "upload_mime_illegal"

All credits to @Kostelano . I only took over his closed pr https://github.com/joomla/joomla-cms/pull/39196

Closes https://github.com/joomla/joomla-cms/issues/38280

### Summary of Changes
- Remove field `upload_mime_illegal` from `com_media` settings.
- Deprecate associated language strings.

### Testing Instructions
- Code review shoud be sufficient after several confirmations that this field is not used in core. See comments: https://github.com/joomla/joomla-cms/issues/38280 and https://github.com/joomla/joomla-cms/pull/39196
- I you think it's worth it: Change/Add mime entries in `com_media` configuration in field `Forbidden MIME Types`.
  - Check if these settings have any influence on file uploads in the Media Manager. This should not be the case.

### Actual result BEFORE applying this Pull Request
- Settings ignored on file uploads in media manager.

### Expected result AFTER applying this Pull Request
- No change. Settings ignored on file uploads in media manager.

### Link to documentations
Please select:
